build.gradle

```diff
@@ -1931,7 +1931,9 @@ project(':storage') {
     implementation project(':storage:storage-api')
     implementation project(':server-common')
     implementation project(':clients')
-    implementation libs.caffeine
+    implementation(libs.caffeine) {
+      exclude group: 'org.checkerframework', module: 'checker-qual'
```

Comment:
It seems `error_prone_annotations` is used in compile time, right? If so, should it be excluded too?

Reply:
@chia7712 , we've discussed `error_prone_annotations` above. In short, it'll fail compilation if exclude it.